cfy_manager/components/haveged/haveged.py

```diff
@@ -11,8 +11,13 @@ class Haveged(BaseComponent):
     component_name = HAVEGED
     services = ['haveged']
 
+    def configure_supervisord_config(self):
+        if self.service_type == 'supervisord':
```

Comment:
plz move the `if` up, into `configure`, because right now if you just read `configure`, it looks like it always configures supervisord, even when using systemd